oic.r.acl2.raml

```diff
@@ -20,6 +20,24 @@ traits:
   get:
     description: |
       Retrieves the ACL data.
+      When used without query parameters, all the ACE entries are returned.
+      When used with an aceid or resources query parameter, only the
+      ACEs matching the specified parameter are returned.
+    queryParameters:
+      aceid:
+          type: integer
+          description: |
+            Returns ACE with the specificed aceid
+          required: false
+          repeat: false
+          example: 2112
+      resources:
+          type: string
+          description: |
+            Returns ACEs with the specificed resource href
+          required: false
+          repeat: false
+          example: coaps://IP-ADDR/temp
```

Comment:
I'm not sure where this example URI came from. It's used all over the place. But I'm happy to change it...